bin/initialize.js

```diff
@@ -21,30 +21,30 @@ program.version(packageVersion);
  * command for initializing a project
  */
 program
-	.command("init [projectname]")
+	.command("init [projectname] [gitrepository]")
 	.alias("i")
 	.description("initialize React project")
 	.option("-l, --eslint", "eslint required or not ?")
-	.action(function(projectname, options) {
+	.action(function(projectname, gitrepository, options) {
 		if (projectname === undefined) {
 			console.log("provide a project name");
 			return;
 		} else {
 			const spinner = ora("creating directory structure").start();
-			init.initialize(projectname, options.eslint, function initProject(
-				res
-			) {
+			init.initialize(projectname, gitrepository, options.eslint,
+				function initProject(res) {
+					console.log('res', res);
 				if (res) {
 					setTimeout(() => {
```

Comment:
@isuruAb Spinner animation needs continue till download is complete in case of a remote template, if it's default template it should stop once it copies default template to specified directory.